modules/theme-tools/featured-content.php

```diff
@@ -493,14 +493,14 @@ public static function customize_register( $wp_customize ) {
 			'priority'       => 20,
 		) );
 		$wp_customize->add_control( 'featured-content[hide-tag]', array(
-			'label'          => __( 'Hide tag from displaying in post meta and tag clouds.', 'jetpack' ),
+			'label'          => __( 'Hide tag from displaying in post details and tag clouds.', 'jetpack' ),
 			'section'        => 'featured_content',
 			'theme_supports' => 'featured-content',
 			'type'           => 'checkbox',
 			'priority'       => 30,
 		) );
 		$wp_customize->add_control( 'featured-content[show-all]', array(
-			'label'          => __( 'Display tag content in all listings.', 'jetpack' ),
+			'label'          => __( 'Display tagged posts outside the Featured Content area.', 'jetpack' ),
```

Comment:
Maybe have this include an "as well" or an "also" at the end?  To indicate that it is both?

![moana-clip](https://cloud.githubusercontent.com/assets/941023/24012861/ebc8e568-0a55-11e7-87a5-d744dcf7b11b.jpg)
(Maui is a hero to all!)